translation-tips.md

```diff
@@ -34,21 +34,21 @@ More specific translation instructions are available for the following projects:
 - [Documentation and website for Kovri (anonymous router)](https://github.com/monero-ecosystem/monero-translations/blob/master/translate-kovri-website.md)
 - [Monerujo (Android wallet)](https://github.com/monero-ecosystem/monero-translations/blob/master/translate-monerujo.md)
 
-Other Monero-related projects, created by community members, are collected in the [Monero Ecosystem](https://github.com/monero-ecosystem/meta).
+Other Monero-related projects, created by community members, are to be found in the [Monero Ecosystem](https://github.com/monero-ecosystem/meta).
 
-We have a localization platform called Pootle, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it only supports the GUI wallet, but will be soon available also for the CLI wallet and Monerujo.
-For now, the default workflow of all projects currently on Pootle, is entering the translations in a text editor and making a pull request in Git or on Github to upload your work.
+We have a localization platform called Weblate, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it only supports the website, Community, GUI and CLI wallet, but will soon also be available for Monerujo, the Android wallet.

```

Comment:
```suggestion
We have a localization platform called Weblate, reachable at [translate.getmonero.org](https://translate.getmonero.org). For now it supports the website, community material, GUI and CLI wallet.

```

There are no plans to include Monerujo at this time, so that part can be removed.